src/js/player.js

```diff
@@ -4703,7 +4706,7 @@ class Player extends Component {
    *                                        from the TextTrackList and HtmlTrackElementList
    *                                        after a source change
    *
-   * @return { import('./tracks/html-track-element').default }
+   * @return {HtmlTrackElement}
```

Comment:
actually makes the jsdoc more informative here https://deploy-preview-8723--videojs-docs.netlify.app/player#addRemoteTextTrack vs https://videojs-docs.netlify.app/player#addRemoteTextTrack